requirements_test.txt

```diff
@@ -9,3 +9,4 @@ six
 # Type packages for mypy
 types-pkg_resources==0.1.3
 tox>=3
+pre-commit
```

Comment:
Can we leave this out? `pre-commit` is not necessary to run the tests.

Reply:
I don't really use `tox`, didn't know we still recommend it.

You can probably just use the CI for this? That's what I always do :smile:

Reply:
Moved `pre-commit` dependency directly to the tox.ini config - does it seem like better solution?